### ISSUE TITLE
Blank page after automatic reload

### ISSUE THREAD

**Issue by combeenator** [aurelia/cli#687]
**I'm submitting a bug report**

* **Library Version:**
CLI `0.30.1`
Framework `1.1.4`

**Please tell us about your environment:**
* **Operating System:**
Windows 10

* **Node Version:**
6.9.4

* **NPM Version:**
4.1.1

* **Browser:**
Chrome 60 but I guess it happens in other browsers as well

* **Language:**
TypeScript 2.4.2

**Current behavior:**
From time to time my page stays blank after an automatic reload due to changes in the source code. If this happens I have to stop & restart the `watch` task which I've started with `au run --env my-env --watch`. 

After the restart everything is back to normal for some time.

Until now I wasn't able to figure out what might causes the issue but can only tell that it happens very frequently (~ once every 5 - 10 minutes). Needless to say that having to restart the task that often is very distracting...

When this happens, my network tab shows me that the `vendor-bundle.js` is loaded but my `app-bundle.js` which usually follows afterwards is not.

Please let me know if I can give you any more information. I'd be really happy if we could somehow eliminate this issue!

**Comment by JeroenVinke** [comment]
Hey @combeenator,

I wonder, what are the sizes of your bundles?

**Comment by combeenator** [comment]
Right now the `vendor-bundle.js` is 1.5 mb and the `app-bundle.js` is 112 kb unminified & unzipped.

**Comment by JeroenVinke** [comment]
Is your project setup similar to a default CLI project? In other words, do you think it would be possible to reproduce in a new cli project?

**Comment by combeenator** [comment]
I've tweaked the default CLI project a lot which makes it most probably very hard to reproduce.

What I could do is "dumb down" my project to the bare minimum where this happens but since it only happens every xx mins it would take a lot of time to do so (dumb down -> test -> dumb down -> test, ...).

Is there anything I can check when it happens next time like debugging something within `vendor-bundle.js` in order to find out "how far we get"?

**Comment by JeroenVinke** [comment]
Yeah that's going to be difficult. I have a feeling that this might be related to https://github.com/aurelia/cli/issues/685, although i'm not sure. Once I know i'll get back to you.  If that turns out to be a different issue, would it be possible for me to run your application so I can debug it?

**Comment by combeenator** [comment]
It's a commercial project which depends on our servers to run but I can try to remove those dependencies and critical code before handing it over to you ;)

Obviously those preparations would also take some time but if we can't find no other way, I'd be happy to provide such a version.

**Comment by JeroenVinke** [comment]
@combeenator I don't think that that issue is related. Mind if I send you a private message on Gitter?

**Comment by suamikim** [comment]
@JeroenVinke Sure, contact me any time!

**Comment by JeroenVinke** [comment]
@suamikim I've sent a message to @suamikim on Gitter
